project/build.gradle.kts

```diff
@@ -3,10 +3,20 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 plugins {
 	id("org.springframework.boot") version "2.2.6.RELEASE"
 	id("io.spring.dependency-management") version "1.0.9.RELEASE"
+	id("io.gitlab.arturbosch.detekt") version "1.0.0.RC6-4"
 	kotlin("jvm") version "1.3.71"
 	kotlin("plugin.spring") version "1.3.71"
 }
 
+detekt {
```

Comment:
detekt is not related to the themes of this branch. It should have been added in another branch.

Reply:
And it was but the branch was deleted and the issue closed, it would be pointless to reopen the issue and do all the process again.

Reply:
It wasn't approved though. 😅